sn_networking/src/cmd.rs

```diff
@@ -372,7 +400,9 @@ impl SwarmDriver {
                     .put_verified(record)
```

Comment:
Can we pass in the `record_type` to `recordstore::put_verified()` too? We're computing the same value inside `put_verified`.